templates/compose/chibisafe.yaml

```
@@ -0,0 +1,78 @@
+# documentation: https://chibisafe.app/docs/intro
+# slogan: A beautiful and performant vault to save all your files in the cloud.
+# tags: storage,file-sharing,upload,sharing
+# logo: svgs/chibisafe.svg
+# port: 80
+
+services:
+  chibisafe:
+    image: chibisafe/chibisafe:v6.5.5
+    environment:
+      - BASE_API_URL=http://chibisafe_server:8000
+    healthcheck:
+      test: ["CMD", "wget", "--spider", "--quiet", "http://127.0.0.1:8001"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 10s
+
+  chibisafe_server:
+    image: chibisafe/chibisafe-server:v6.5.5
+    volumes:
+      - database:/app/database:rw
+      - uploads:/app/uploads:rw
+      - logs:/app/logs:rw
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "wget",
+          "--spider",
+          "--quiet",
+          "http://127.0.0.1:8000/api/health",
+        ]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 10s
+
+  caddy:
+    image: caddy:2-alpine
+    environment:
+      - SERVICE_FQDN_CHIBISAFE_80
```

SERVICE_FQDN variable gives value without http protocol so we have to use SERVICE_URL
```suggestion
      - SERVICE_URL_CHIBISAFE_80
```

---

Ah - the template was made before coolify made this change:

- https://github.com/coollabsio/coolify/commit/e8892b3d29290b69b796a6f2bddeab84d323b392

which switched those 2.